suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -147,6 +210,70 @@ const discoverAccountsByDescriptorThunk = createThunk(
     },
 );
 
+export const addNetworkAccountThunk = createThunk(
+    `${DISCOVERY_MODULE_PREFIX}/addNetworkAccountThunk`,
+    async (
+        {
+            network,
+            accountType,
+            deviceState,
+        }: {
+            network: Network;
+            accountType: AccountType;
+            deviceState: string;
+        },
+        { dispatch, getState },
+    ): Promise<AddAccountResult> => {
+        const accounts = selectDeviceAccounts(getState()).filter(
+            account => account.symbol === network.symbol && account.accountType === accountType,
+        );
+
+        const index = accounts.length + 1;
+
+        // only 10 accounts per network/accountType are supported
```

Comment:
I hope there is some constant for it that can be used here.

Reply:
4b55b435df